packages/@o3r/workspace/schematics/sdk/index.ts

```diff
@@ -47,8 +47,9 @@ function generateSdkFn(options: NgGenerateSdkSchema): Rule {
     ]), MergeStrategy.Overwrite);
 
     const packageManager = getPackageManager({ workspaceConfig });
-    const specExtension = options.specPackagePath ? path.extname(options.specPackagePath) : '.yml';
-    const specPath = options.specPackageName ? `openapi${specExtension}` : options.specPath;
+    const specExtension = options.specPackagePath ? path.extname(options.specPackagePath) : '.yaml';
+    // TODO: Change `swagger-spec` to `openapi` in v11
```

Comment:
can we create an issue so that we don't forget about it?

Reply:
updated